scheduling/pulse.py

```diff
@@ -0,0 +1,15 @@
+import os
+from pyspark import SparkConf, SparkContext
+from pyspark.sql import SQLContext
+from python_etl.testpilot import pulse
+
+if __name__ == "__main__":
+    conf = SparkConf().setAppName('python_etl')
+    sc = SparkContext(conf=conf)
+    sqlContext = SQLContext(sc)
+
+    # Make bdist_egg available to executer nodes
+    sc.addPyFile(os.path.join('dist', os.listdir('dist')[0]))
```

Comment:
Non-blocking for this review, but would we be able to distribute this library by default like `python_moztelemetry` in the airflow clusters to avoid specifying these files?